composer.json

```diff
@@ -34,7 +34,7 @@
         "doctrine/lexer": "^2",
         "doctrine/persistence": "^2.4 || ^3",
         "psr/cache": "^1 || ^2 || ^3",
-        "symfony/console": "^4.2 || ^5.0 || ^6.0 || ^7.0",
+        "symfony/console": "^5.0 || ^6.0 || ^7.0",
```

Comment:
I guess this line (and for symfony/yaml) can be changed to `^5.4 || ^6.0 || ^7.0` to only support LTS Versions?

Reply:
yeah I guess